src/index.css

```diff
@@ -104,6 +105,70 @@ h1 {
   margin-top: 8px;
 }
 
+nav {
+  background-color: #262626;
+  width: 100%;
+  position: sticky;
+  top: 0;
+  height: auto;
+}
+
+nav li {
+  font-size: 19px;
+  font-weight: 700;
+  line-height: 26px;
+  text-align: center;
+  margin-right: 16px;
+  margin-bottom: 8px;
+  margin-left: 16px;
```

Comment:
How about padding left & right on the `ul` or `nav`? Then you don't need to put margin left & right on the `li`?

I mean, I guess this is a question of preference in the end and it doesn't really matter because the result is the same. But it "feels" more fitting to put padding on the container instead of margin on every item.